lib/module.js

```diff
@@ -123,6 +130,27 @@ function axiosModule (_moduleOptions) {
 
   logger.debug(`baseURL: ${options.baseURL}`)
   logger.debug(`browserBaseURL: ${options.browserBaseURL}`)
+
+  // Create a separate axios instance on req.$axios for server middleware usage.
+  this.nuxt.hook('render:setupMiddleware', (app) => {
+    app.use((req, res, next) => {
+      const runtimeConfig = {
+        ...this.options.publicPrivateConfig,
+        ...this.options.privateRuntimeConfig
+      }
```

Comment:
Is there a better way to access runtime config from a module? Ideally with resolved values...